frontend/src/containers/Faq.js

```diff
@@ -28,40 +28,40 @@ export class Faq extends Component {
           <h1>FAQ</h1>
 
           <h3>What is an Open Collective?</h3>
-          <p>An Open Collective is formed by a group of people who collect and spend money together to do things in full transparency: everyone who adds funds to the collective can follow the money and see all the transactions.</p>
+          <p>An Open Collective is a group of people with a shared mission that operates in full transparency. Everyone can see how they receive or spend money.</p>
 
           <h3>What can I do with OpenCollective?</h3>
           <ul>
-            <li>Set up a page to collect membership fees or donations for your group</li>
-            <li>Let your members submit expenses and reimburse them with a click</li>
-            <li>Give everyone visibility on your budget</li>
+            <li>Set up a page to collect membership fees or donations (<a href="https://opencollective.com/RailsGirlsAtl">like this one</a>)</li>
+            <li>Let your members submit expenses and reimburse them with a click (<a href="https://opencollective.com/RailsGirlsAtl/expenses/new">example</a>)</li>
+            <li>Give everyone visibility on your budget (<a href="https://opencollective.com/RailsGirlsAtl#budget">example</a>)</li>
           </ul>
-          <p><em>If you have an organization, you can use OpenCollective to empower local chapters to raise money locally and have their own budget. See <a href="#organizations">OpenCollective For Organizations</a>.</em></p>
+          <p>If you have an organization, you can use OpenCollective to empower local chapters to raise money locally and have their own budget. See <a href="#organizations">OpenCollective For Organizations</a>.</p>
 
           <h3>What is OpenCollective good for?</h3>
-          <p>OpenCollective is perfect for any project where you don’t want the hassle of creating a legal entity before getting to the fun stuff. It’s ideal for meetups, open source projects, parent associations, neighborhood associations, pet projects, clubs, unions, movements, non-profits, business incubators, etc.</p>
+          <p>OpenCollective is perfect for any side project that is not for profit. Side projects should be fun. You shouldn't have to worry about creating and maintaining a legal entity, doing bookkeeping, etc. as soon as you need to collect money to reimburse your expenses. OpenCollective is ideal for meetups, open source projects, parent associations, neighborhood associations, clubs, unions, movements, non-profits, business incubators, etc.</p>
```

Comment:
Are we sticking with OpenCollective or Open Collective? I personally prefer latter. It's different below.